### PR TITLE
[Skin_default] add more buttons

### PR DESCRIPTION
**Pull request by AbuBaniaz**
png files taken from 720 skin, (PLI HD)

### CODE REVIEW

**Comment by littlesat** [comment]
Is this still required due to the other previous change

**Comment by AbuBaniaz** [comment]
Yes, otherwise no PNG files will be found.

File: data/skin_default/buttons/Makefile.am [hunk 1]
```diff
@@ -23,19 +24,35 @@ dist_install_DATA = \
...
+	key_text.png \
+	key_tv.png \
+	key_yellow.png \
+	left.png \
+	play.png \
+	rec.png \
 	red-big.png \
 	red.png \
+	repeat-off.png \
+	repeat_on.png \
```
Comment (LraiZer) [inline data/skin_default/buttons/Makefile.am]: repeat-on.png has hyphen, not underscore ;p

Comment (AbuBaniaz) [reply data/skin_default/buttons/Makefile.am]: Thanks. Corrected.

However, would be better if wildcards were used like on other images

**Pull request merged**